tests/unittests/cityscapes_tests.py

```
@@ -0,0 +1,26 @@
+
+import fiftyone.utils.cityscapes as foucs
+
+
+class CityscapesSplitTests(unittest.TestCase):
+    def test_parse_split(self):
+        self.assertEqual(foucs._parse_split("train"), "train")
+        self.assertEqual(foucs._parse_split("train_extra"), "train_extra")
+        self.assertEqual(foucs._parse_split("validation"), "val")
+        self.assertEqual(foucs._parse_split("test"), "test")
+
+        with self.assertRaises(ValueError):
```

Ruff (0.15.17)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)